index.html

```diff
@@ -53,6 +53,120 @@ <h1>I’m Seko<br> Glad to <br>see you!</h1>
+                        <li class="btn btn-secondary">html</li>
+                        <li class="btn btn-secondary">css</li>
+                        <li class="btn btn-secondary">javascript</li>
+                    </ul>
+                    <button class="btn btn-primary">See Project</button>
+                </div>
+            </div>
+
+            <div class="work-container">
+                <div class="work-image">
+                    <img src="/img/art printing.png" width="278px" height="220px" alt="">
+                </div>
+                <div class="work-content">
+                    <h2>Professional Art</h2>
+                    <ul class="work">
+                        <li class="work-item works-title">Art</li>
+                        <li class="work-item works-role"> <img src="/img/Counter.png" alt=""> Back End Dev</li>
+                        <li class="work-item works-year"> <img src="/img/Counter.png" alt=""> 2015</li>
+                    </ul>
+                    <p>A daily selection of privately personalized reads; no accounts or sign-ups required.</p>
+                    <ul class="techs">
+                        <li class="btn btn-secondary">html</li>
+                        <li class="btn btn-secondary">css</li>
+                        <li class="btn btn-secondary">javascript</li>
+                    </ul>
+                    <button class="btn btn-primary">See Project</button>
+                </div>
+            </div>
+
+
+        <section class="about-section">
+            <div class="about-container">
+                <div class="desc">
+                    <h2 class="about-heading">About<br> Myself</h2>
+                    <p class="about-desc">Hello I’m a software developer! I can help you build a product , feature or website Look through some of my work and experience! If you like what you see and have a project you need coded, don’t hestiate to contact me.</p>
+                    <a href="#" class="connect">Let's Connect</a>
+                    <div class="social-icon">
+                        <ul>
+                            <li><a href="#"><img src="/img/twiter icon.svg" alt="Twitter icon"></a></li>
+                            <li><a href="#"><img src="/img/Linkedin icon.svg" alt="LinkedIn Icon"></a></li>
+                            <li><a href="#"><img src="/img/medium icon.svg" alt="medium icon"></a></li>
+                            <li><a href="#"><img src="/img/github icon.svg" alt="github icon"></a></li>
+                            <li><a href="#"><img src="/img/hi icon.svg" alt="Hi icon"></a></li>
+                        </ul>
+                    </div>
+                    <button class="btn btn-primary">Get My Resume</button>
+                </div>
+                <div class="skils">
+                    <ul class="skills-items">
+                        <li class="skills-item"><h3>Languages</h3> <img src="/img/Union.png" alt=""></li>
```

Comment:
- [ ] kindly add The technology list ("JavasScript", "HTML" ..) in languages section like shown in figma

![image](https://user-images.githubusercontent.com/89282221/191742359-f41ccf0c-1dab-4515-ba9a-1b22b8205358.png)

Reply:
To get that done, I will need to use javascript that we are not supposed to use. form the project requirement page"You don't need to implement the detail modal window that requires JavaScript."

![Screenshot 2022-09-22 135611](https://user-images.githubusercontent.com/33840476/191752852-5fd1257d-8af5-46b7-a87b-529967f3b274.png)

Reply:
Please note that Javascript is not needed to implement this 😄 . You do not necessarily need to collapse the Languages frame, for now it is okay to implement them just as they are in the Figma design 👍 👍